docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
@aliok I don't see that this file is linked anywhere?

Reply:
@laurafitzgerald that is linked in the JSDocs generated. I advise this:

* do the steps to run the AG.org website locally
* go to localhost:4000
* go to Digger page
* click on Digger Node Client specs button
* you will see the JSDoc HTML generated

Reply:
@aliok thanks. I carried out those steps. Just wondering should these files be linked from somewhere. If they are generated I guess JSDocs knows what it's doing. There is however no way for the user to get to this file without manually adding it in the URL. But if we are happy with this then there is no change required.

Reply:
we just create a link to JSDoc's index.html. Other pages for the modules are linked from there.